src/functionalTest/resources/application.yaml

```diff
@@ -5,6 +5,9 @@ scheduling:
       enabled: false
```

Comment:
Can we add some functional tests that call the payment api? Would be beneficial to check that it works on the pipeline